src/shared/ui/form/ErrorMessage.tsx

```diff
@@ -6,7 +6,7 @@ import { useTranslation } from 'react-i18next';
 import { Text, TextProps } from '@shared/ui';
 
 type Props = {
-  error?: FieldError | { message: string };
+  error?: Partial<FieldError>;
```

Comment:
Please double-check if it doesn't break the other error messages that are used in other forms. Especially, when a **backend error** is passed to the component

Reply:
Already tested and working 👍🏽